tests/functional/testBenchmark.js

```diff
@@ -1,14 +1,57 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 const vpn = require('./helper.js');
 const assert = require('assert');
 const { homeScreen, generalElements } = require('./elements.js');
+const constants = require('./constants.js');
```

Comment:
Move this to `setupVpn.js` so that it will work for all the tests.
And also, `setupWasm.js` with the searchParams. For wasm you also need to change `wasm/test.html` to set the env vars.